go.mod

```diff
@@ -28,14 +28,16 @@ require (
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golangci/golangci-lint v1.18.0
-	github.com/google/go-cmp v0.5.2
+	github.com/google/go-cmp v0.5.6
+	github.com/google/gofuzz v1.2.0 // indirect
```

Comment:
Can you check if this is really necessary? When I bump `openshift/api` locally I get this:

```
diff --git a/go.mod b/go.mod
index 50523691..fba6adf5 100644
--- a/go.mod
+++ b/go.mod
@@ -35,7 +35,7 @@ require (
 	github.com/huandu/xstrings v1.2.0 // indirect
 	github.com/imdario/mergo v0.3.9
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/openshift/api v0.0.0-20210409143810-a99ffa1cac67
+	github.com/openshift/api v0.0.0-20210629145910-15a1cae1fca8
 	github.com/openshift/client-go v0.0.0-20210112165513-ebc401615f47
 	github.com/openshift/library-go v0.0.0-20210301154249-aa29957b8a9c
 	github.com/openshift/runtime-utils v0.0.0-20200415173359-c45d4ff3f912
@@ -51,11 +51,11 @@ require (
 	github.com/xeipuuv/gojsonpointer v0.0.0-20190905194746-02993c407bfb // indirect
 	golang.org/x/net v0.0.0-20210224082022-3d97a244fca7
 	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
-	k8s.io/api v0.21.0-rc.0
+	k8s.io/api v0.21.1
 	k8s.io/apiextensions-apiserver v0.21.0-rc.0
-	k8s.io/apimachinery v0.21.0-rc.0
+	k8s.io/apimachinery v0.21.1
 	k8s.io/client-go v0.21.0-rc.0
-	k8s.io/code-generator v0.21.0-rc.0
+	k8s.io/code-generator v0.21.1
 	k8s.io/kubectl v0.21.0-rc.0
 	k8s.io/kubelet v0.21.0-rc.0
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
@@ -67,7 +67,7 @@ replace (
 	github.com/godbus/dbus => github.com/godbus/dbus v0.0.0-20190623212516-8a1682060722
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1
 	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v0.1.2-0.20190408193819-a1b50f621a48
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20210409143810-a99ffa1cac67
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20210629145910-15a1cae1fca8
 	github.com/openshift/cluster-api => github.com/openshift/cluster-api v0.0.0-20191129101638-b09907ac6668
 	github.com/securego/gosec => github.com/securego/gosec v0.0.0-20190709033609-4b59c948083c
 	k8s.io/api => k8s.io/api v0.21.0-rc.0
``

Reply:
+1 please bump only the minimum set of changes

Reply:
`go mod vendor && go mod tidy` from your PR produced this. If this is not what you are looking for, please guide me for the **how**

Reply:
@kikisdeliveryservice thanks a lot for your review, how to get the minimum set of changes, I applied `go mod vendor && go mod tidy` and this was the result

Reply:
> `go mod vendor && go mod tidy` from your PR produced this. If this is not what you are looking for, please guide me for the **how**
cc: @bertinatto

Reply:
@kikisdeliveryservice  @bertinatto fixed 👍🏽

Reply:
once you have updated required deps, you can simply run `make go-deps`

Reply:
@sinnykumari thanks for pointing out `make go-deps`.

I noticed that this target marks some scripts as executable. IMO the vendor directory should not be changed by anything other than `go mod vendor`. I think this should be handled in a different way, like whatever is calling those scripts should pass them as args to `bash/sh` or something else. Anyway, that's up to you folks.

Reply:
these executable mode change is there for a while now, it is a good idea to re-validate and see if we even need it now or not.